sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/View.java

```diff
@@ -496,6 +622,22 @@ public void processElement(@Element T element, OutputReceiver<KV<Void, T>> r) {
     }
   }
 
+  /** An identity {@link PTransform}. */
+  private static class IdentityTransform<T> extends PTransform<PCollection<T>, PCollection<T>> {
```

Comment:
Is this used anywhere?

Reply:
Not any more. It was used in a much earlier version.